NSS-Bashful-Turtles/Yak-Social-Network
# Rm add posts to profile

Added the posts component from news feed to the profile view. Viewing a profile will now also pull in the 5 most recent posts from the user and display them below the header. 

to test 
```
git fetch --all
git checkout rm-add-posts-to-profile
```

1. make sure json server is running our api
1. make sure npm dev server is running
1. load up web page, a user profile is hard coded to load in from index.js
1. make sure everything looks ok and that recent posts are showing up. 
1. change the user profile being passed to the component to see if information changes accordingly.
1. review code
